src/index.html

```diff
@@ -9,8 +9,25 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
+    <script src="chrome-extension://mooikfkahbdckldjjndioackbalphokd/assets/prompt.js"></script>
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#" class="search__form"
+    data-qa="big">
```

Comment:
Need to fix code style everywhere
```suggestion
    <form
      action="#"
      class="search__form"
      data-qa="big"
    >
```